common/utils/src/main/resources/error/error-classes.json

```diff
@@ -92,6 +92,11 @@
     ],
     "sqlState" : "22003"
   },
+  "BIT_POSITION_OUT_OF_RANGE" : {
```

Comment:
IIRC we have an error class for invalid function arguments, shall we reuse it?

Reply:
We have the error class `INVALID_PARAMETER_VALUE`, @panbingkun could you re-use or create a sub-class there, please.

Reply:
Okay, let me to do it.